assets/paymentManager/spec.json

```diff
@@ -2086,6 +2086,26 @@
       },
       "title": "Amount"
     },
+    "TransactionAmount": {
```

Comment:
Could you update the file `assets/paymentManager/README.MD`, in order to track for future reference the reason of this change?

Reply:
Completely missed that changelog, I added an entry here: https://github.com/pagopa/io-app/pull/3963/commits/f8c7a896b50372fbb81091c35331b9e52a31eb26 😃